models/intermediate/int_zendesk__schedule_spine.sql

```diff
@@ -2,7 +2,7 @@
 
 /*
     The purpose of this model is to create a spine of appropriate timezone offsets to use for schedules, as offsets may change due to Daylight Savings.
-    End result will include `valid_from` and `valid_until` columns which we will use downstream to determine which schedule-offset to associate with each ticket (ie standard time vs daylight time)
+    end result will include `valid_from` and `valid_until` columns which we will use downstream to determine which schedule-offset to associate with each ticket (ie standard time vs daylight time)
```

Comment:
Accidental lowercase?

Reply:
Looks like it, thanks for catching! Just updated.